GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/Scenes/SampleScene.unity

```diff
@@ -0,0 +1,188 @@
+%YAML 1.1

```

Comment:
Rename SampleScene to meaningful names

Reply:
Also use CamelCase for the other scene -> PlayScene

Reply:
Actually Sample scene is empty. I will delete it.